src/dapps/types.ts

```diff
@@ -2,6 +2,8 @@ export enum DappSection {
   RecentlyUsed = 'recently used',
   Featured = 'featured',
   All = 'all',
+  FavoritesDappScreen = 'favorites dapp screen',
+  FavoritesHomeScreen = 'favorites home screen',
```

Comment:
What's the difference between these 2?

Reply:
Oh ok, I guess the screen is already part of the other props attached, but doesn't hurt to add this explicitely.